benchmark_runner/common/ocp_resources/create_nhc_snr.py

```diff
@@ -26,16 +26,15 @@ def create_nhc_snr(self):
             logger.info(f'run {resource}')
             self.__oc.create_async(yaml=os.path.join(self.__path, resource))
 
-            if '03_subscription.yaml' in resource:
-                # verify once after create all resource files
+            if '03_nhc_subscription.yaml' in resource:
```

Comment:
For the future, we should not have to hard code all of this inline in this way.  But that's a separate issue and not a problem with this PR per se.